react-select/react-select-tests.tsx

```diff
@@ -12,8 +12,10 @@ class SelectTest extends React.Component<React.Props<{}>, {}> {
 
     render() {
         const options: ReactSelect.Option[] = [{ label: "Foo", value: "bar" }]
+        const onOpen: () => void = () => {}
+        const onClose: () => void = () => {}
```

Comment:
the implementation doesn't match the type... it returns `{}`, not `void` (inline arrow functions implicit return). if you implement the function correctly then you shouldn't need to explicitly type the variable.